pkg/kubernetes/run.go

```diff
@@ -41,6 +46,21 @@ func (o *RunOptions) Validate(args []string, cfg config.Config) error {
 	return nil
 }
 
+//TODO: implement our own Serve until porter/pkg/plugins supports protocolVersion
```

Comment:
I've opened an issue to track making this change back in Porter https://github.com/getporter/porter/issues/1986